src/main.zig

```diff
@@ -2005,6 +2014,16 @@ fn buildOutputType(
         else => false,
     };
 
+    // Always output import libraries (.lib) when building for msvc to replicate
+    // `link` behavior. lld does not always output import libraries so on the
```

Comment:
Actually, we always output import lib regardless of the ABI, so for mingw too. Hence why I suggested always setting the default for `out_implib`. This way we would unify interfaces between `build-lib` and `cc`, which I believe we should aim to do.